lib/ui/useable-components/google-maps/location-bounds-profile-restaurants/index.tsx

```
@@ -0,0 +1,814 @@
+                            parts.map((part, index) => (
+                              <span
+                                key={index}
+                                style={{
+                                  fontWeight: part.highlight ? 700 : 400,
+                                  color: 'black',
+                                  marginRight: '2px',
+                                }}
+                              >
+                                {part.text}
+                              </span>
+                            ))}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)